src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -9,6 +9,7 @@
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
     <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.7'">$(DefineConstants);netstandard17</DefineConstants>
+    <NugetTargetMoniker Condition="'$(TargetGroup)'=='netstandard1.7' AND '$(NugetTargetMoniker)'==''">.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
If TargetGroup is netstandard1.7 I wouldn't ever expect NugetTargetMoniker to be empty. See https://github.com/dotnet/corefx/blob/dev/api/dir.props#L363.